app/routes/project/posts/post.js

```diff
@@ -1,17 +1,43 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  session: Ember.inject.service(),
+
   model(params) {
     let projectParams = this.paramsFor('project');
-
-    return this.store.queryRecord('post', {
+    let queryParams = {
       sluggedRouteSlug: projectParams.sluggedRouteSlug,
       projectSlug: projectParams.projectSlug,
       number: params.number
+    };
+
+    let userId = this.get('session.session.authenticated.user_id');
+
+    return Ember.RSVP.hash({
+      post: this.store.queryRecord('post', queryParams),
+      user: Ember.isPresent(userId) ? this.store.find('user', userId) : null
+    }).then((result) => {
+      return Ember.RSVP.hash({
+        post: result.post,
+        comment: this.store.createRecord('comment', { post: result.post, user: result.user }),
+        comments: this.store.query('comment', { postId: result.post.id })
+      });
```

Comment:
Model is returning an `RSVP.hash` because, based on documentation I'm finding, it should make it the simplest for us to switch to routed components once those are out.